src/datatable/__init__.py

```diff
@@ -52,13 +52,13 @@
     setdiff,
     shift,
     sort,
+    split_into_nhot_depr as split_into_nhot,
```

Comment:
@st-pasha After your changes, `split_into_nhot_depr` should be changed to `split_into_nhot_deprecated` here.

Reply:
ok